app/views/kiosks/index.html.erb

```diff
@@ -25,7 +25,11 @@
             <td><%= link_to kiosk.name, kiosk %></td>
             <td>
               <% if kiosk.restart_at_active == true %>
-                <span class="label label-danger">Pending</span>
+                <% if kiosk.restart_at > DateTime.now || kiosk.restart_at > 1.minutes.ago %>
```

Comment:
This would block in the cell can be condensed to 

```
<% if kiosk.should_restart? %>
  <span class="label label-danger">Pending</span>
<% else %>
  <span class="label label-default">Restarted</span>
<% end %>
```